### PR TITLE
adding aws ec2 fake data

### PR DESCRIPTION
**Pull request by joshdvir**

### CODE REVIEW

File: test/test_aws.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  def test_instance_type
+    assert_match(/^[a-z]{1}[0-9]{1}\.[a-z]+/, FFaker::AWS.instance_type)
+  end
+
+  def test_offering_type
+    assert_match(/\A[ a-zA-Z]+\z/i, FFaker::AWS.offering_type)
+  end
+
+  def test_instance_tenancy
+    assert_match(/\A[default|dedicated|host]+\z/i, FFaker::AWS.instance_tenancy)
```
Comment (marocchino) [inline test/test_aws.rb]: It should be  `/\A(?:default|dedicated|host)\z/`

File: test/test_aws.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+require 'helper'
+
+class TestAWS < Test::Unit::TestCase
+  include DeterministicHelper
+
+  assert_methods_are_deterministic(FFaker::AWS, :instance_type)
+
+  def test_instance_type
+    assert_match(/^[a-z]{1}[0-9]{1}\.[a-z]+/, FFaker::AWS.instance_type)
```
Comment (marocchino) [inline test/test_aws.rb]: This test is broken. 
My suggestion is `/\A[a-z]+\d\.\d*[a-z]+\z/`.

Comment (joshdvir) [inline test/test_aws.rb]: fixed.

File: test/test_aws.rb (hunk 1 above)
Comment (joshdvir) [inline test/test_aws.rb]: fixed

**Comment by marocchino** [comment]
Thanks!

File: lib/ffaker/data/aws/ec2_reserved_offering_type [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Heavy Utilization
+Medium Utilization
+Light Utilization
```
Comment (kesor) [inline lib/ffaker/data/aws/ec2_reserved_offering_type]: These heavy/medium/light utilization types no longer exist in AWS. New reserved intances are always "heavy" utilization, with their different types of offering - No/Partial/All Upfront 1year, Partial/All Upfront 3year and Convertable No/Partial/All Upfront 3year.

Comment (joshdvir) [inline lib/ffaker/data/aws/ec2_reserved_offering_type]: it's backward compatible, also there is and offer type there you have the offerings

**Pull request merged**